resume.md

```diff
@@ -1,49 +1,34 @@
-# FIRST LAST NAME
+# Niharika Desaraju 
 
 (123) 456 - 7890 | email@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/ndesaraju | linkedin.com/in/niharika-desaraju
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of California, Berkeley** | B.S. Bioengineering, Minor Computer Science May 2021
+* **Coursework**: Data Structures, Algorithms, Machine Learning, Discrete Math, Probability, Computational Biology, Databases, Computer Architecture, Computer Security.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python, Java, C, SQL, Swift
+* **Libraries/Frameworks**: Pandas, TensorFlow, RESTful APIs, Node.JS, Flask, Numpy, HealthKit, ResearchKit
+* **Version Control**: Git
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Research Intern**: UCSF - Henry Roland Laboratory | August 2018-Present
+* ● Built a secure Apache HTTPS Server to create a dynamic web application for physicians and researchers to query for and visualize volumetric data.
+* ● Utilized Google Compute Instances to construct a high performance computing infrastructure  to process over 7,000 C2-C3 and C3-C4 PSIR images collected from clinical trials. This pipeline segments the spinal cord and extracts volumetric data that will be used to examine  the correlation between spinal cord volume and progression of MS.
```

Comment:
Talk more about the technicalities of using Google Compute Instances.  There is a lot of jargon in this bullet point that a recruiter might not understand.  Why does this data matter?